Podfile

```diff
@@ -0,0 +1,5 @@
+target 'sampleapp-tvos-swift' do
```

Comment:
We should specify the min TV OS SDK version in the Podfile too, I think it's 10.0